libdnf/base/base_impl.hpp

```diff
@@ -67,6 +81,7 @@ class Base::Impl {
 class InternalBaseUser {
 public:
     static solv::Pool & get_pool(const libdnf::BaseWeakPtr & base) { return base->p_impl->get_pool(); }
+    static solv::Pool & get_comps_pool(const libdnf::BaseWeakPtr & base) { return base->p_impl->get_comps_pool(); }
```

Comment:
In DNF4 we used DnfSack for modules and different instance for RPMs. It created a confusion in code what data I am accessing. What about to use a different Pool class for groups?

Reply:
Hm, both pools use exactly the same data structure, thus they are instances of the same `solv::Pool` class. I do not like adding some `CompsPool` subclass of `Pool` class without any change.
What is different for both pools is contents - and this is expressed by variable / function name.